src/extension.ts

```diff
@@ -92,7 +92,7 @@ async function upCommand(selectedManifestUri: vscode.Uri) {
     } catch (err) {
         reporter.track(events.manifestLoadFailed);
         reporter.captureError(`load manifest failed: ${err.message}`, err);
-        return onOktetoFailed(`Okteto: Up failed to load your Okteto manifest: ${err.message}`);
+        return onOktetoFailed('', `Okteto: Up failed to load your Okteto manifest: ${err.message}`);
```

Comment:
Instead of passing around an empty string, make the parameter optional in `onOktetoFailed` and give it a default value. I'd rather use 'null' to denote the default terminal for these cases instead of an empty string.